src/lib/posenetWrapper.js

```diff
@@ -1,63 +1,101 @@
+      rightElbow: 8,
+      leftWrist: 9,
+      rightWrist: 10,
+      leftHip: 11,
+      rightHip: 12,
+      leftKnee: 13,
+      rightKnee: 14,
+      leftAnkle: 15,
+      rightAnkle: 16
     };
   }
 
-  bodypart(bodypart){
-    return (new Bodypart(this.source['keypoints'][this._bodypartIndexLookup[bodypart]]))
+  bodypart(bodypart) {
+    return new Bodypart(
+      this.source["keypoints"][this._bodypartIndexLookup[bodypart]]
+    );
   }
 
-  isMountainPose(){
+  isMountainPose() {
     // var conditional =  new conditional([
     //this._isPointBetween(blah),
     //this._isHigherThan(blah),
     //this._isLowerThan(blah),
     //])
     // return conditional.isMet
-    const c1 = this.bodypart("rightWrist").position['y'] > this.bodypart("rightHip").position['y']
-    const c2 = this.bodypart("leftWrist").position['y'] > this.bodypart("leftHip").position['y']
+    const c1 =
+      this.bodypart("rightWrist").position["y"] >
+      this.bodypart("rightHip").position["y"];
+    const c2 =
+      this.bodypart("leftWrist").position["y"] >
+      this.bodypart("leftHip").position["y"];
     // lower body x-coordinates
-    const c3 = this._isPointBetween(this.bodypart("leftAnkle").position['x'],[this.bodypart("leftShoulder").position['x'],this.bodypart("rightShoulder").position['x']])
-    const c4 = this._isPointBetween(this.bodypart("rightAnkle").position['x'],[this.bodypart("leftShoulder").position['x'],this.bodypart("rightShoulder").position['x']])
-    const c5 = this.bodypart("leftAnkle").position['x'] > this.bodypart("rightAnkle").position['x']
-    const c6 = this._isStraight([this.bodypart("leftAnkle").position,
-                                this.bodypart("leftKnee").position,
-                                this.bodypart("leftHip").position])
-    console.log("is straight:")
-    console.log(c6)
-    return (c1 && c2 && c3 && c4 && c5 )
+    const c3 = this._isPointBetween(this.bodypart("leftAnkle").position["x"], [
+      this.bodypart("leftShoulder").position["x"],
+      this.bodypart("rightShoulder").position["x"]
+    ]);
+    const c4 = this._isPointBetween(this.bodypart("rightAnkle").position["x"], [
+      this.bodypart("leftShoulder").position["x"],
+      this.bodypart("rightShoulder").position["x"]
+    ]);
+    const c5 =
+      this.bodypart("leftAnkle").position["x"] >
+      this.bodypart("rightAnkle").position["x"];
+    const c6 = this._isStraight([
+      this.bodypart("leftAnkle").position,
+      this.bodypart("leftKnee").position,
+      this.bodypart("leftHip").position
+    ]);
+    console.log("is straight:");
+    console.log(c6);
+    return c1 && c2 && c3 && c4 && c5;
   }
 
-  _isPointBetween(point,boundary){
-    boundary.sort(function(a,b){return a - b});
-    const withinUpperBound = boundary[1] > point
-    const withinLowerBound = boundary[0] < point
-    return withinUpperBound && withinLowerBound
+  isTreePose() {
+    const c1 =
+      this.bodypart("rightAnkle").position["y"] >
```

Comment:
So here you're just checking that neither foot is raised above the respective knee, right?